app/controllers/version.js

```diff
@@ -18,8 +19,22 @@ export default Controller.extend({
   }),
 
   actions: {
-    selectVersion(version) {
-      this.transitionToRoute('version', version)
+    selectVersion(newVersion) {
+      const currentPath = getOwner(this).lookup('router:main').currentURL;
+      // url format is /version/section/page/
+      const [ , , sectionTitle, pageTitle] = currentPath.split('/');
+
+      this.store.query('page', { version: newVersion }).then(sections => {
```

Comment:
I can't seem to find any detailed documentation on the `filter` syntax.
The JSON API standard just says that it's up to the server implementation (which I'm not familiar with in this project).
http://jsonapi.org/format/#fetching-filtering
The Ember guides/API docs don't do anything as complex as only getting a single record where an array property contains an object that has an attribute with a specific value:
https://guides.emberjs.com/release/models/finding-records/
https://www.emberjs.com/api/ember-data/release/classes/DS.Store/methods/query?anchor=queryRecord
https://www.emberjs.com/api/ember-data/release/classes/DS.Store/methods/query?anchor=query

My guess is that it would look something like this:
```javascript
      this.store.queryRecord('page', {
        filter: {
          version: newVersion
          pages: {
            filter: { 
              url: `${sectionTitle}/${pageTitle}`
            }
          }
        }
    })
```
Do you have an example of the type of query I could make here?

Reply:
Thanks for your review @toddjordan :)

Reply:
if you look at the application adapter: https://github.com/ember-learn/guides-app/blob/master/app/adapters/application.js#L7-L8

you'll see that queryRecord expects a "path" argument.  You should be able to match that with a json file in guides source.